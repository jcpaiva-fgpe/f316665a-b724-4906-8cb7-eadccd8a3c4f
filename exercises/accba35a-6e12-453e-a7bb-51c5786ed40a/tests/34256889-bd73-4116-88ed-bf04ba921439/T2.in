22
1 1 1
-1 2 3
3 -1 2
4 4 -1
0 0 0
3 3 0
-8 -8 -8
5 5 6
10 34 32
32 10 34
34 32 10
-4 -8 -7
0 0 -40
0 3 3
0 3 0
7 10 24
7 24 10
24 10 7
7 24 10
10 7 10
24 7 10
8 8 8
