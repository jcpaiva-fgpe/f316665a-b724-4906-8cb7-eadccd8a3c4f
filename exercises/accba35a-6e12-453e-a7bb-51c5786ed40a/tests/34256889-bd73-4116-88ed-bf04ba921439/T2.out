Sim, equilatero.
Claro que nao!
Claro que nao!
Claro que nao!
Claro que nao!
Claro que nao!
Claro que nao!
Sim.
Sim.
Sim.
Sim.
Claro que nao!
Claro que nao!
Claro que nao!
Claro que nao!
Claro que nao!
Claro que nao!
Claro que nao!
Claro que nao!
Sim.
Claro que nao!
Sim, equilatero.
